tests/filecheck/mlir-conversion/with-mlir/dialects/llvm/llvm_types.mlir

```diff
@@ -0,0 +1,26 @@
+// RUN: mlir-opt %s --mlir-print-op-generic | xdsl-opt --print-op-generic | filecheck %s
+
+"builtin.module"() ({
+  // CHECK:   "builtin.module"() ({
+
+  // Type tests
+
+  // literal
+  %s0, %s1 = "test.op"() : () -> (!llvm.struct<()>, !llvm.struct<(i32)>)
```

Comment:
Wait you can pass test ops through mlir? Without allowing unregistered dialects? I am confused :thinking:

Reply:
The test seems to pass, so I guess it works 🤷

Reply:
Hmm weird. We should investigate that.

Maybe add an issue? I think we can merge this meanwhile.

Reply:
#1018